control-plane/p4RuntimeArchStandard.h

```diff
@@ -418,6 +421,47 @@ template<> struct ActionSelectorTraits<Arch::PNA> : public ActionProfileTraits<A
     }
 };
 
+// Traits for the MatchValueLookupTable extern
+template<Arch arch> struct MatchValueLookupTableTraits;
+
+template<> struct MatchValueLookupTableTraits<Arch::PNA> {
+    static const cstring name() { return "emvlt"; }
+    static const cstring typeName() {
+        return "MatchValueLookupTable";
+    }
+    static const cstring sizeParamName() {
+        return "size";
+    }
+};
+template<> struct MatchValueLookupTableTraits<Arch::PSA> {
+    static const cstring name() { return "emvlt"; }
+    static const cstring typeName() {
+        return "MatchValueLookupTable";
+    }
+    static const cstring sizeParamName() {
+        return "size";
+    }
+};
+
+template<> struct MatchValueLookupTableTraits<Arch::V1MODEL2020> {
```

Comment:
This may be just PNA specific. Why do we add it for V1Model

Reply:
it's easiest to fix compilation rather separating again entire p4info generation mechanism for pna from rest.
anyway it will be compilation error for  rest of all arches

Reply:
ok